batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java

```diff
@@ -1016,7 +1016,7 @@
 
     /**
      * Lists the execution status of the Job Preparation and Job Release task for the specified job across the compute nodes where the job has run.
-     * This API returns the Job Preparation and Job Release task status on all compute nodes that have run the Job Preparation or Job Release task. This includes nodes which have since been removed from the pool. If this API is invoked on a job which has no Job Preparation or Job Release task, the Batch service returns HTTP status code 409 (Conflict) with an error code of JobPreparationTaskNotSpecified.
+     * This API returns the Job Preparation and Job Release task status on all compute nodes that have run the Job Preparation or Job Release task. This includes nodes which have since been removed from the pool. If this API is invoked on a job which has no Job Preparation or Job Release task, the Batch service returns HTTP status code 409 (Conflict) with an error code of JOB_PREPARATION_TASK_NOT_SPECIFIED.
```

Comment:
The comment is generated by swagger spec. If you change this way, eventually, we have to manually update the comment every time we update swagger spec.

Reply:
Yeah agreed, for the same reason, these changes were reverted. This specific version is outdated.
Latest version no longer has these changes: https://github.com/g2vinay/azure-sdk-for-java/blob/batch-sdk-integration/batch/data-plane/src/main/java/com/microsoft/azure/batch/protocol/Jobs.java

Reply:
Latest changes can be seen here: https://github.com/Azure/azure-sdk-for-java/pull/2865/files